src/Aquifer.Common/Clients/SendGridClient.cs

```diff
@@ -1,49 +0,0 @@
-using SendGrid;
```

Comment:
I moved the `SendGridClient` to the Aquifer.Jobs project and pretty much rewrote it.  Now only the `EmailSubscriber` is allowed to talk to SendGrid and the `EmailService` will put emails onto a queue.